DeepNeuralNetworks/DNN.ipynb

```diff
@@ -1,1399 +1,2548 @@
 {
```

Comment:
<p>This is a really good visual, but some reasoning on why this happens would be helpful. We've already talked about under and over training but an explanation of in and out of domain learning is also helpful </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>